samples/java/src/main/java/Endorser.java

```diff
@@ -85,7 +88,7 @@ static void demo() throws Exception {
         String schemaAttributes = new JSONArray().put("name").put("age").put("sex").put("height").toString();
         AnoncredsResults.IssuerCreateSchemaResult createSchemaResult =
                 issuerCreateSchema(authorDid, schemaName, schemaVersion, schemaAttributes).get();
-        String schemaId = createSchemaResult.getSchemaId();
+        // String schemaId = createSchemaResult.getSchemaId();
```

Comment:
Would it be better to remove this variable since it's not used?

Reply:
Done